### ISSUE TITLE
🐞 - [Ionic] Incorrect popover position on iOS emulator

### ISSUE THREAD

**Issue by sahiljain2497** [hakimio/ngx-ui-tour#119]
### Which ngx-ui-tour-* package is the source of the bug?

ionic

### Description

We have implemented the tour plugin on this page and it works on web but when we test it on IOS simulator it isn't showing on correct positions. 
I have attached the video to compare both web and ios. we are calling the startTour function from ngAfterViewInit function.

Here is base code to run this  - 

added 

```
<ion-app>
  <ion-router-outlet id="main-content"></ion-router-outlet>
  <tour-step-template></tour-step-template>
</ion-app>
```

```
startTour() {
    this.tourService.initialize(
      [
        {
          anchorId: "portfolioDiv",
          content: "Manage your portfolio here and add funds to trade",
          title: "",
        },
        {
          anchorId: "cat-btns",
          content: "Manage your all tokens from these quick actions",
          title: "",
        },
        {
          anchorId: "tokens-table",
          content: "Check out some popular token trending in market",
          title: "",
        },
        {
          anchorId: "niftyscore",
          content:
            "Your NiftyScore is deter- mined by latest trade price multiplied by number of tokens in circulation. Everyone gets a score as soon as their 1st token is traded and will change over time",
          title: "",
        },
      ],
      {
        showArrow: true,
        disablePageScrolling: true,
        enableBackdrop: true,
        smoothScroll: true,
        placement: { alignment: "center" },
      }
    );

    this.tourService.start();
    this.tourService.end$.subscribe({
      next: (res) => {
        this.onTourCompleteOrSkip();
      },
    });
  }
```

https://user-images.githubusercontent.com/28824550/232802227-6eb11fea-2727-4e88-9c2b-5778068e6b93.mov

https://user-images.githubusercontent.com/28824550/232802235-f301d29b-50aa-40e3-9708-15fcd4f1f36b.mp4

### Angular version

15

### Ngx UI Tour version

1.6.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [X] iOS
- [ ] Android

**Comment by sahiljain2497** [comment]
I have tried above changes but no luck. Tested on real device as well but it has the same issues.

**Comment by hakimio** [comment]
Try `smoothScroll: false`

**Comment by sahiljain2497** [comment]
Still same, web and ios device
![localhost_8100_tabs_tab3(Pixel 5)](https://user-images.githubusercontent.com/28824550/233050383-193dcb77-8fa7-4faf-8be6-5c768ab8e5fb.png)

![IMG_2760](https://user-images.githubusercontent.com/28824550/233050537-fe0deec7-d1ad-469f-9ea8-31d4eadd6f98.PNG)

**Comment by hakimio** [comment]
Does this only happen on iOS or is Android effected as well?

**Comment by hakimio** [comment]
Also, does this happen on macOS Safari?

**Comment by sahiljain2497** [comment]
> Does this only happen on iOS or is Android effected as well?

Working on android

> Also, does this happen on macOS Safari?

Working on macOS safari

**Comment by hakimio** [comment]
Ok, it's difficult for me to debug this since I don't have Mac or iPhone.
Anyway, my guess would be that the tour tries to show the step before the page is fully rendered and the tour gets incorrect anchor element position. It's strange that setting `disablePageScrolling: false` and `delayBeforeStepShow: 1e3` doesn't fix it.

**Comment by hakimio** [comment]
Also, do you have `[fullscreen]="true"` set on your `ion-content`?

**Comment by sahiljain2497** [comment]
> Also, do you have `[fullscreen]="true"` set on your `ion-content`?

no, I don't have this attribute

**Comment by sahiljain2497** [comment]
> scroll-padding



> BTW, did you set `scroll-margin` or `scroll-padding` CSS property? Is the incorrect offset the same as your header height?

I don't have these properties setup. But, yes the offset seems like the same as maybe the header or tab size below

**Comment by sahiljain2497** [comment]
No, ion-popover has the same issue. it has same height difference

**Comment by hakimio** [comment]
Then the issue might be specific to your application.
Can you reproduce it with one of the Ionic starter/demo apps?

**Comment by sahiljain2497** [comment]
ok going to try that, is there a way I can just center these for IOS?

**Comment by hakimio** [comment]
Tour step centering is described in [the FAQ](https://hakimio.github.io/ngx-ui-tour/ion-popover/FAQ#center-tour-step).

**Comment by hakimio** [comment]
Closing since it seems to be Ionic issue.

**Comment by hakimio** [comment]
The ionic popover bug might to be fixed in [v7.3.3](https://github.com/ionic-team/ionic-framework/releases/tag/v7.3.3).
